app/assets/stylesheets/thredded/components/_post.scss

```diff
@@ -4,6 +4,71 @@
   @include thredded-media-mobile {
     margin-bottom: $thredded-base-spacing;
   }
+  .thredded--post--dropbtn .thredded--post--three-dot-menu {
+    fill: currentColor;
+    color: $thredded-action-color;
+    display: inline-block;
+    font-size: 1em;
+    width: 1rem;
+    height: 1rem;
+    padding-left: 1.5rem;
+  }
+
+  .thredded--post--dropdown {
+    position: relative;
+    display: inline-block;
+    float: right;
+    text-align: right;
+    cursor: pointer;
+    @media print {
+      display: none;
+    }
+  }
+
+  .thredded--post--actions {
+    display: none;
+    position: absolute;
+    right: 0;
+    background-color: #f9f9f9;
```

Comment:
This doesn't work with the dark theme. Let's add a `$thredded-overlay-background` variable, and set the default to